component/addons/additional-rules.libsonnet

```
@@ -0,0 +1,28 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local prom = import 'lib/prom.libsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.prometheus;
+{
+  prometheus+: {
+    prometheusRule+: {
+      spec+: {
+        groups+: [
+          {
+            name: group_name,
+            rules: [
+              local rnamekey = std.splitLimit(rname, ':', 1);
```

Might make sense to error here if the key is malformed:

```suggestion
              local rnamekey =
                local k = std.splitLimit(rname, ':', 1);
                assert std.member([ 'alert', 'record' ], k[0]) : 'Invalid custom rule key "%s", the component expects that custom rule keys are prefixed with either "alert:" or "record:"' % [ rname ];
                k;
```